generate/gogen/types.go

```diff
@@ -186,7 +188,6 @@ func (g *generator) convertType(juteType parser.Type) (*goType, error) {
 		}
 		return &goType{
 			typeID:    typeClass,
-			isPtr:     true,
```

Comment:
Is the `isPtr` attribute true in any case? Should it be removed?

Reply:
After these changes, it's always false. Removed the field and its usages.